yaml_edit/lib/src/editor.dart

```diff
@@ -158,14 +157,17 @@ class YamlEditor {
   ///
   /// If [orElse] is omitted, it defaults to throwing a [ArgumentError].
   ///
-  /// To get `null` when [path] does not point to a value in the [YamlNode]-tree,
-  /// simply pass `orElse: () => null`.
+  /// To get a default value when [path] does not point to a value in the
+  /// [YamlNode]-tree, simply pass `orElse: () => [YamlNode]`.
```

Comment:
```suggestion
  /// [YamlNode]-tree, simply pass `orElse: () => ...`.
```

nit: source links don't work in code-snippets (or so I think)..